lib/atspi/atspi_node.cc

```diff
@@ -362,82 +370,80 @@ std::vector<AtspiNode> AtspiNode::getChildren() const {
 // interface. See https://gitlab.gnome.org/GNOME/at-spi2-core/-/issues/167 for
 // details.
 AtspiActionInterface AtspiNode::queryAction() const {
-  AtspiAction* iface = atspi_accessible_get_action_iface(accessible_);
-  if (iface) {
-    g_object_unref(iface);
-    return AtspiActionInterface(ATSPI_ACTION(accessible_));
+  ScopedGObjectPtr<AtspiAction> iface(
+      atspi_accessible_get_action_iface(accessible_.get()));
+  if (iface.get()) {
+    return AtspiActionInterface(iface.release());
   }
 
   return AtspiActionInterface();
 }
 
 AtspiComponentInterface AtspiNode::queryComponent() const {
-  AtspiComponent* iface = atspi_accessible_get_component_iface(accessible_);
-  if (iface) {
-    g_object_unref(iface);
-    return AtspiComponentInterface(ATSPI_COMPONENT(accessible_));
+  ScopedGObjectPtr<AtspiComponent> iface(
+      atspi_accessible_get_component_iface(accessible_.get()));
+  if (iface.get()) {
+    return AtspiComponentInterface(iface.release());
```

Comment:
I was trying to find out what `release` does here -- but as I commended on ScopedGObjectPtr, I can't find where "release" is defined. Does it do something other than just returning a raw ptr?

Reply:
Answered below.